cortex-js/src/app.ts

```diff
@@ -23,7 +23,7 @@ export const getApp = async () => {
   const config = new DocumentBuilder()
     .setTitle('Cortex API')
     .setDescription(
-      'Cortex API provides a command-line interface (CLI) for seamless interaction with large language models (LLMs). Fully compatible with the [OpenAI API](https://platform.openai.com/docs/api-reference), it enables straightforward command execution and management of LLM interactions.',
+      'Cortex API provides a command-line interface (CLI) for seamless interaction with large language models (LLMs). It is fully compatible with the [OpenAI API](https://platform.openai.com/docs/api-reference) and enables straightforward command execution and management of LLM interactions. Cortex = `http://localhost:1337/api`. Jan = `http://localhost:1337/v1`',
```

Comment:
Please help rebase, I think the file is relocated

Reply:
@louis-jan please check, Am I doing the rebase correctly?